src/org/daisy/dotify/formatter/impl/page/PageSequenceBuilder2.java

```diff
@@ -298,24 +300,47 @@ private PageImpl nextPageInner(
             boolean wasSplitInSequence,
             boolean isFirst
     ) throws PaginatorException, RestartPaginationException { // pagination must be restarted in
-        // PageStructBuilder.paginateInner
+                                                              // PageStructBuilder.paginateInner
         PageImpl current = newPage(pageNumberOffset);
-        if (
-            pcbl != null &&
-            // Store a mapping from the BlockLineLocation of the last line of the page before the
-            // previous page to the BlockLineLocation of the last line of the previous page. This
-            // info is used (in the next iteration) in SheetDataSource to obtain info about the
-            // verso page of a sheet when we are on a recto page of that sheet.
+        // whether this is the last page of the volume and a volume transition was provided
+        boolean interruptContentPresent =
+            context.getTransitionBuilder().getProperties().getApplicationRange() != ApplicationRange.NONE &&
             transitionContent.isPresent() &&
-            transitionContent.get().getType() == TransitionContent.Type.INTERRUPT
```

Comment:
Please fix the checkstyle errors.

`/home/travis/build/mtmse/dotify.formatter.impl/src/org/daisy/dotify/formatter/impl/page/PageSequenceBuilder2.java:24:8: Unused import`

`import org.daisy.dotify.formatter.impl.core.TransitionContent.Type;` should be removed

Reply:
Fixed in https://github.com/bertfrees/dotify.library/commit/137db797379da574c1ebf51aee1189771c63f28d.